Framework/PythonInterface/test/testhelpers/__init__.py

```diff
@@ -38,7 +40,11 @@ def create_algorithm(name, **kwargs):
         kwargs - A dictionary of property name:value pairs
     @returns The algorithm handle
     """
-    alg = mantid.api.AlgorithmManager.createUnmanaged(name)
+    if 'Version' in kwargs:
```

Comment:
Slightly more pythonic, and using knowledge of `AlgorithmManager` is to change this to
```python
alg = mantid.api.AlgorithmManager.createUnmanaged(name, kwargs.get('Version', -1))
if 'Version' in kwargs:
    del kwargs['Version']